test/object_test.rb

```diff
@@ -51,10 +57,27 @@ class ObjectTest < MiniTest::Spec
       end
     end
 
-    it do
+    it "transforms the object using representer" do
       representer.prepare(source).to_object
-      _(source.album.name).must_equal "Live"
-      _(source.album.songs[0].title).must_equal 1
+
+      _(source.album.name).must_equal("Live")
+      _(source.album.songs[0].title).must_equal(1)
+    end
+
+    describe "when used as decorator" do
+      representer!(name: :decorator ,module: Representable::Object, decorator: true) do
+        property :name, getter: ->(represented:, **) {
+          represented.name = "#{represented.name} is the best team of America"
+        }
```

Comment:
Why is the `getter` setting a value?

Reply:
Hi @apotonick,

I just changed the test to use the same logic in the existing test, but using `Representable::Object` as a decorator. 

As an honest answer to your question: I just pushed that commit and add the comment to be able to get some help to understand what is the goal of `to_object`. It looked wrong to set values from a lambda passed in an option called `getter`. But I didn't know which option makes sense when using a decorator to call `to_object`

I created this PR to be able to use from `from_object` with allows me to instantiate an object base on a source one. 

```
[3] pry(#<#to_object::when used as decorator>)> source = Album.new("Atletico Nacional")
=> #<struct ObjectTest::Album name="Atletico Nacional", songs=nil>
[4] pry(#<#to_object::when used as decorator>)> new_object = decorator.prepare(Album.new).from_object(source)
=> #<struct ObjectTest::Album name="Atletico Nacional", songs=nil>
[5] pry(#<#to_object::when used as decorator>)> new_object.object_id
=> 1760
[6] pry(#<#to_object::when used as decorator>)> source.object_id
=> 1780
[7] pry(#<#to_object::when used as decorator>)>
``` 

But, according to what I understand, `to_object` does modify the passed object.